src/trading/base.py

```
@@ -66,42 +136,260 @@ def to_dict(self) -> dict[str, str]:
+        creator=creator or user,
+        creator_vault=creator_vault,
+        **kwargs
+    )
 
-    def _get_relevant_accounts(self, token_info: TokenInfo) -> list[Pubkey]:
-        """
-        Get the list of accounts relevant for calculating the priority fee.
 
-        Args:
-            token_info: Token information for the buy/sell operation.
+def create_lets_bonk_token_info(
```

Ruff (0.12.2)

Too many arguments in function definition (9 > 5)

(PLR0913)